### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by TroyKomodo**
## Proposed changes

PR adds better documentation

## Types of changes

What types of changes does your code introduce to Scuffle?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/ScuffleTV/scuffle/blob/main/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: design/README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+Scuffle can be broken into a few different components:
 
-They provide a better understanding of how complex systems interact with each other.
+## [Video](./video/README.md)
 
-- [High Level Design](./high-level.md) For a high level overview of the project.
-- [Video Infrastructure](./video-infrastructure.md) For some specific design on video infrastructure.
-- [CDN Edge](./cdn-edge.md) CDN Edge mainly for video edge.
-- [Database](./database.md) For database design.
+Scuffle Video is a live streaming as a service product (SAAS/PAAS). Scuffle Video has a API which allows for creation of rooms and has various configuration options to control how transcoding and recording works. This component is designed to be self hosted and can be deployed to a kubernetes cluster, or single server or whatever you want. This is a very flexible component and comes with a video player which can be embedded into any website. This is not a Scuffle Platform specific component and you do not need to use the streaming platform component to use the video component.
```
Comment (lennartkloock) [inline design/README.md:7]: Some typos I found

```suggestion
Scuffle Video is a live streaming as a service product (SAAS/PAAS). Scuffle Video has an API which allows for creation of rooms and has various configuration options to control how transcoding and recording works. This component is designed to be self hosted and can be deployed to a kubernetes cluster, single server or whatever you want. This is a very flexible component and comes with a video player which can be embedded into any website. This is not a Scuffle Platform specific component and you do not need to use the streaming platform component to use the video component.
```

**Review by mmattbtw — APPROVED** [review]
seemsgood

**Review by SputNikPlop — APPROVED** [review]
This seems fine, I don't think it is well balanced between being detailed and having simplicity.

**Comment by TroyKomodo** [comment]
> This seems fine, I don't think it is well balanced between being detailed and having simplicity.

Is there any improvements i can make to make them better?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,12 +4,100 @@
...
+
+Commits must be in the format specified here https://karma-runner.github.io/6.4/dev/git-commit-msg.html
+
+`doc(api): Added documentation for xyz`
+
+where the format is basically
+
+`type(scope): <description>`
+
+Then in the commit message body, you can give a more detailed description and link to the ticket that this commit aims to resolve.
```
Comment (SputNikPlop) [inline CONTRIBUTING.md:45]:
```suggestion
Then in the commit message body, you can give a more detailed description and link to the ticket that the commit aims to resolve.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,12 +4,100 @@
...
+`abc` is no longer supported and has been replaced with `xyz`
+```
+
+If you have any questions regarding how commit messages should be formatted please ask.
+
+## Pull Requests
+
+Each commit in a pull request should resolve one or more tickets.
+
+There should be one commit per ticket. If we need more tickets then we can create sub-issues and tasks around those.
```
Comment (SputNikPlop) [inline CONTRIBUTING.md:63]: I think we should be more clear here it sounds like it could be worded better.

**Review by treuks — APPROVED** [review]

**Review by 4w3Official — APPROVED** [review]

**Pull request merged**